### ISSUE TITLE
LongConversion not checked when generating parselet for e.g. periodicUpdate(long)

### ISSUE THREAD

**Issue by JerryShea** [OpenHFT/Chronicle-Wire#157]

**Comment by RobAustin** [comment]
@JerryShea what do you mean by "not checked", please give more details on the issue.

**Comment by JerryShea** [comment]
```
    void periodicUpdate(@LongConversion(MicroTimestampLongConverter.class) long currentTimeMicros);
```
The @LongConversion is not used when generating a parselet for the above i.e. you have to write
```
perioidicUpdate: 1523983274829
```
in your test - you can't use a date string

**Comment by JerryShea** [comment]
Fixed in #183

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.101](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
